tsfc/driver.py

```diff
@@ -398,9 +414,18 @@ def compile_expression_dual_evaluation(expression, to_element, *,
             try:
                 expr, point_set = expr_cache[pts]
             except KeyError:
-                point_set = PointSet(pts)
+                if runtime_point_interpolation:
+                    assert isnan(pts).all()
```

Comment:
add comment, either here or at the top when getting the fiat equivalent - "Until FInAT dual evaluation is done, FIAT QuadratureElements with UnknownPointSingleton point sets advertise NaNs as their points for each node in the dual basis.